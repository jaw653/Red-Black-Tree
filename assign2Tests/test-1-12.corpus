zy	dfl
mf
	 riwfgz 
yg	kw
yf 
zl 
dlrs	
b	h
tz		 
rvhmcpbg 
roefxo

b
 uirzapwa
ey
yq t
b
rz	mwc ba
i		 
	
ga
	 
isoh 

amul	dacx
iluszmulm
i lfg	pov
bluvyif
p

bie		 g	
hnthugrad
wtpa a	yr
va uh xyv
 	
xcnydw
vt	
lorrl
d  sk

mk
y r hbn
g
yumvj uo	
o
  
pjia
v puzx rb

	o	

o 

rd fcml
p
hf
wfi
r

i pwg 
	l
j t	czqt	

oicdu t

ii gn	 		
q
imn	 bf
 mekouuyd
h	qr
 a
f

	sx	nepp
q t	njzdq
q
o
w	ce 
 
nzyx	x 
wivbodbhx
	
m	 
h	g
cyrkate o
x
kadey d
lickv	
rb
	gu
	ty	n
		oi 
tl 
d
wrt
 st
wkgx sh		
	bbj
f
az
czbnyu	f 
yhgrx	xcc
rb dhhdr	
ye	j
m	o

odx	usl
	
a		t
eexg
d
	ms
l  


	c g fm

l
 d np 
j	f
 sb	b
sn	b	
l


e	fyooi  
	ofy
z	rb



ng unj
	qjf
 	a 
rw	envoby
p		

  fp
ayiv ibv


r kf	csl
	
xfhuy 

rtmv
hta

	c pnnq
q
z	k	f	m	s
l
o
fhd	 
zxt	mrtsy
	fxaqjc	b
j	
q 
aqq
hvuspjewl
	ze	ol
bg
pwrefcnj 
	olvtpuy

	a
fwnl g
	wabaf 
x
x
	kwn	
l
h	guun

 
td ps
dp	
bdhhyp
fs
ss
xwxsos
h
oeur	jp

psw	oq
	
zo tbd au
o
	 tfz f
 	a	lw
z 
	mplpsb


x
hhzyhun
		nxxfxbb
dw
wq af 
j
s
pt	zq
ble 	v	xf
vlhn i
ho
	h
o ag
p
hxmgehm	c
rbas	azjc
d

pym
qq
b	bhbhw b
okhyou		

edemrjl j
k	 vbwpkf
jy		nphyv
le
 remnk
fs		 
 li
mpah
u
jm
	l 
 	
x 

mna
db 


 idxwvo

	fwk msug
fc
	mjp	w

djwfbbfo
 q	
nrqbi
	
yq cq v
xthv			ry
pymbve my
r	awwttwf

r	 	z
ly
n o

	xi	
 k 	shjnw

		w
	exr
a
cl dir	
     u p	
	
s 


y

j	pbzdszd

qmoetjau
hme
	or
o
y

 jizzk

ubdi luf
pyl	wqmyj
h
koeys
j
ae aimp
j
d 

w ucz

cw
	k
  
	sqlnpcf

gqduibj
p
x
 		rq
 
udda sr c
etbq 	
ni
b	yuft o

nkywiaetg
 ee	w
j
z
zl
j	a
gn
c	 ook t

vevjt
 qh
l
twlregh
 	mz wyos
	ard  g	m
q
gfl	mq 
k	
bky fj
yy
		ol	 
knzf
	jh	
ig
 qar 	
xv sg	 rv
t 
	
mh  
i	
dppu	u
agy o  
i
 ck 
 ne

 
ght kzu
	  cxcc	s

e
k
	i	u
b ei	r		w
m la k 
j
h
k ql
el
q	yanw hy
l

unsey

  	np

b 
is
	k r k



g lpph
 
noqdyuh
qng
ruld

nsam 	c
l
g
kc

lk 
 ojd	pgt 
g
  knwmx
pdzrb kyb
	d 	jrvkv
	
rkutx
	
g	jaop x 
	 		bcog 
l		oexf	c
rp bvgi  
lgtyocj  
rq r
nm b
xc
zxz	i	
ja ib  w 
yvbbv	w
g
	p  
iq
d
sr	
	vwx	
	yzlvij  
  
g
f	cz
ykqyr 
z 
zc	 l 	vv
lx 
u	 e 
zg
v d
 

wwn m	i c
h
ir
m	  
i
en 
	
m
l p

	zlw
	
		k	qwy
df
afih	u

 o	g pd 
cr
 yse	n

 m 	ujmv
o w l	c	g

 	rdhp	o
ri	e
h	lg


awcmhk	
abyoww rf
stzs
ztfr
ggvs	b		

l		rui	b


oemdtaid
d fuqydk

srj cya
r

l		 dam


 s

	ulx
 q	kqboh

 	vul
 


kor  r
dl
	sfwbro e
vm
s j
nx
v
aphvvj	
	k
jdvr	o
uko vxf h
	

gi	
xn
	g
qjpcco

uj	ljft

jsr
jz 


xmzmg
b	w
pnpor
d
z
 ms

qw i
yzh
	xi c
rmj
ies		
rb	
i	htr
jtnbm  r	

ywwbskbk
 e	kyh
y	
t 	cxt	ft
pu
 f
ebe
zwqtghwra
	
k jfstw
djdniqhu	
p	qo	jl
f

qhrzt	qr
enmvl bu	
 aebwz
m 

np		dek	
gefa

	oq
mu
m	y 	c
x
erl	a	q
l foca	s 
ry yuk	c

	r	

lov


be g
ypy
m syw	odw
wqgjdn	lg
		pq	sn


ei
  czdf

  		ahbt
wb	ko
qst
	kx okiye
myr	
 pm	
ao	g doz	
	ilmew
hw
kzd	sdxuk
rkbrmftfr
xlla	x
	

	mfqmmhnx
c
u
	twoj
j	 zyt	ay
e
bgu		sq
im
nk
smv
	
 woqmi


	a ok 	s
ji  cbng	
	
bkd
s	

 ivhvu	
 
j joqrg	v
zj	by 
ui
x

t jmrw
uw 
jdb p
j ch
scq

f dq
vsin
fet
	fmm

	i 	p	
m

ux


c	yh
nf y	sgrg
 aco
	roq
gryri
 	p
		
k		
gq
efrgjsi	m
l
hl l	bm

	w	 edqf
	r sln vt
l em  	hq
wtdg
  fx
xr
 	m
em
br	ftv	v

sg	gxc jw



zfy	u


	vlhayom

s
qn 
	r
meqc	
crm
t o
gajtu
 apa  uoq
xu	hi	tt	
avlbmznlb
zimuk	gfj
	sti	 y c
	unjsdc


c	 tgnbsq
lx	hlp	i

nnrss 
r 
w 
qct	cd
goe	 a 
	
pcugvr
mj
k a
g
g h
xp	e
clsp
q bfi	r
y
j ujn	bfz
t
w	 m f 
dqb


y	z
pjxfoi d 
he h tlsh
pjdhh 	v	
etsr 	tu 
eu
xsp  h
k
dynb
w	
x
rik
z 

 e
ij		
r
o 
	 s	cz
	
vwxjpp

		cmlihjz
 a wwtqud

gigpswnc
d	n  cvdx
hhwc		hgm
ah
wt sze
h f	q
	aw
zdr r
p 	
bwhpwfguv
edvotxs
r
pz
g	owpz
v
  	zwh	
 e
dwyy  
  
wpq
pf
bqgpkv	v	
oc	n	e	
t
u l
kzc q

dr
z	qqw
w 
eur	tr
 
qqbq
	g
o
h
	
w s
k	x	nv	n	
ucjpnf
if
r 	


kn 
rt	ohpwke
q
	l	 ek 
hb	sxwwk	
rzjiv

	 

amt tuds

lvao x


	
 
 w
se
de	cmqb
a
bvar	 q	u
t 
czb
	x
fgy
l xl	
gq hbiykt
ast flbk 

			o	xqy

qg		ehpw
fod u
  y
zj ss	 ap
af nkh	a 
g qy st	v
d i	xy	sf
 lssljyvj
tlau
zw s
stbrxn
xf
g	o
x

go
 yx
q h
t
 bqwme zm

	ox
n	
g
q	rponatb
pieuvmhh 
atyr	x		d
u

smffe

g
cniljgk
bdcj	ox	

r	
dtobje
sse
p	
c	
ihrwcpdvp
jc		 c
g	
ps	
 	fgt
i y af h 
p
glz s	y
gqh pfqfx
tb
rqkast
	
	u	zis	
peziw
h	

	i jxbp	j

yifzn
 t
xagwm 
in
wndw l i	
hsg	l pxm

 
n 	juj

z
tm zl


aqitcy	h
	o
s
e
dk
h

mw	
	b
t 
i bkfa

ncew
 fd
nlro
qbci
	sa 		
	l
 u
hp
	a

ez	
xatwk
l
h
olb y
	sgg gq	p

cuezaf	f
od ju
k	c
lmo	ny og
n	kbxas k
q
 p

j x
qpzzqvikz
lb	m
		
v
y	
fuae t
yv
w	keg 
 
jhm	m h
ack	
rteq
e	
w
i

	
 gszhkg q
j  z x	ic
qnnorrm e
d	
o
mt
v
oo
zm
dl 
qug
a

j

g  r		
pf
 
djv	jnn
k	r
p t
i
xyi	 u	eq
js l a 


 
 fau	jh
 d
 k akp
zg jehlj	
	
qu	wqg	
b		isg
g	
s rmh dhx

y	h	z	d



p
iq	  
uf p
aa	

n m t  ao
vgyab
dpv
nb dquhak

isyq
ia 
z
z 		l	 
kfl	fpj
r
		q	m cq	
 nbvk
dy 
hrd
coydf
   	d	pmy
mg l

mb	
fpcg
 kk	
	wpgq	gjs
p

yo z


h	fix

re
	ykhqhfc

 ersu	qsm
u nsmz			
yk
	ce
wk
unxah 

	


da  
	o

 
	hovyh
joi ne o 
e
rqp kuf

	ssj p	d
	hp hbx	

txaph zg	
jdc
lf  o
mxip 
biu
r
lkuvzld

d t k te
i	v ljoz

	ui
	cl e
vhfap yog
e 
zp
dzj
nauq xxik
 piu
xbgw
	r	
chpwh
gprbl
upe
klad	aqzh
o po 	w	c
  		bsb r
fk	luqtj

l n
qeiod

	qdsi
ld
dpwhk kna
gb 	u z	e
v msfmm
e
 
j uqquw

	ib	ibt 
xw	opa sc
vn
qtpzq	

hvdytww

ga jsdk t
mt gwd	 g
h  	cw q

t  bs	 sh

jr
kj		t
yywbp
nkf
owgs
 	v 
tj	i
	l j
	op  k	hw
	ax mz wl
bkn ah ol
b   gc	nm
maeap	 o	
b
pg	w
sp
		ty	apk

lsj
s	opm
 xh	ih 	u
z	sl
dcm	
	x
	a iu 
du v i
lc
	 	 t
zh

aan 
nq
v
mqhcgg n 
 elj
we	


gp	cchu


ew pbb	n

mx rcs
n
w	gaq 

c
astpgxpa

wo
o
y
je
k
 mxkudf
td r  t


duip k	eb
qv
tm
ede
slazkxssq
p s	whijm
e
f	pz	t

io mz axb

g

vdmcf
pyqm y  e
	n
smb
	c

mla


 n
	lbfqlh	d
gxxf c  a
	 	dt ujn
g	nqzkx
y
 a	dphvng
cgus	cbqe
rxr
dua
c
	 bpqiqus
z

 mxl 

 paello
k
 a	irla	l
vovprdcru

dsvn gbo
	k
	fwvg

xx

wlciv
xg
z xb	c
l	m
j h
p
 omswitso

tf 

pmp
vw			 er 
maa fu
jk
nobr phvw
t
pi
ic	

 qmhmubfb
p gsp
 g 
ax euo
tx
	m	vwgodo
 ll  p 	 
	p dw
tn

	 	b mv
c
vorqhim
b
cs	zf 	rh
  g
zg	nz
v	kmo

tg
z
wxadf
e
ysq 	l
hd
r 	bl o


oany
nn 	

muiou	j	
 ys	n ovn
br
c	  qw

 	roztrh
 mc	o l  
	qhanfrav
fw axq
	o
yj i	pbwk
tcy	v
	aw
w		ny u
t
n
di
ft
v
fk
cgi l	
a
kptand 
ku
weasxn
yzu	uatww
lm
	 x
u	
egf qpleo

	uezqbvr
o
pim
x


hm swnc
k
 y pk	mal


b trp
q
ie i
	dgz
sdqikgs	o
aacaef
h

mgf	e  
	
pxyo	m t 
a	mbhovoq
cnt
zl
gj
jmi

kqjc
 	qus	x


cxn	i		bu
h	cqm
w	p
	
	ysjexs
u
s		y  	
	a	a
yron
va	
iesv	
suul	 k	 
p
mrr	wfr
xxpnk u
b
	ciruxj f

 s ffly

zgneykuen
	fah	 kgx

m rwa	p

cb dxt
rn
 in fvuxp
mhwz 
 nu

erd	alch
pm	p
l   
l
py
	vcm

vyxtpzrd

kv ounwm
hn iwx
yh

		bsybw 
qepn	a	eo
po	qwmtw 
cgbfvwxkt
gtz	w	st 
bhvs  kgz
fabdsrqoe
nrmbtx dq
x
wdhgkto
hzbpmexin
bv 	 vac 
f
icv
fu	
hnr
sh
ew
tvsljv ic
qwqx	flno
f srsom
 
wt
r			fe
		
blqs u
k jkdjb
c
rvxuk

s 
dpw bvfid
 b	v	
 ru
urn
z 	sr
nb
	c f w
m
n	c	bsp
  	mfej	o
n pnqpu
 
 d nqkgq	
dxbtx

fw
siog	e u	
 ad	y 	pp
muacs
		


u
 k rmh
lepok 	i 

	
j
eogp

q	wkd
x	

zu	d	js

f  ttym		
a		
 e
jw
heuu	dr 

g	
p	d
ef
p	ej
fmnn
oqdols  s
vy
yv	 i	


nq	ehh

fr		 m  n
 y	tz

zh
 	gufx
l 
 en
xn
f	
ann

uj n
w 
	pobnj
la q aph 
okwiz
d o
x
xgg
	
	
vdh 	 
rb
nln 	ldn	
  ock
ct	
wteztd

m
ryznyqhhh
qbde	vi  
	haqi	  k
bmoujhnc	
za rdf	qz
ftltngc e

fsqnpeif
t 
jnckwe

ylyj aqq
yzc	gkcp

 phzq d v
z
iy esjn
  	gsjsv

	t	
lmr
a

p tcdvuy
xh  uaxcz
	
y
aztzb
	f
y e	cw
r feyb ed
s om q ut



 
y ct
i
o clfej
q	l	ju
wt
	rn   kwj
	k m	
zzs
d	bdn	e		
kmk	
eef	
yvbprkipi
v o
seasr
krbg 	
bt
wscezxzf 

 murij	b
vp u	a 
w
	ges

js

r ohkmeji
	
   f  

di g	zjsn
ig g 
	m 
i	vvbxqmd
 	z
trj		
 z

nwy
o
h  		i	 	
w mg		fl

bi
kn	
lp
zoh qa	ss
bsjytth
	
 ocvbgq
z
jf	 rsx v
	
	qxgpkq
ffr 
rwmm
habi	zgbc
xug 	bhsc

eh
cq	sn
 cyr  xp 
udf	kbztc
zc					ne
e	 
wmm l
f
smopa 	
lo xoajh 


 h
	kt	
z

k
eo	q
bxgftd
z	
x	eydzxks
mdut
qvow
bbfefnngm
	ex i 	sz

 yl qumq
	 	yoqyun
oz		
ctmc
yewf 
	
	
sr	q
c
 

k
or	ihg 
d	vfe	nqj
k alhu	d

			
bkx c
w	javw 
 

v
grc
jh
seu wr
v

om
sj
ctm
let	shqz

tz  x	zil
fpgodd
eu
xl 	
m vt
 el
z whu
 r fjfa f
and
c
zz

j	svitnc 
caofn rqh
 
en jb
	
eja	hy p

flko	c g 

	el w 
v
j	pttpueb
e kxp	afz
lnqued
gu

	oyxf	
n
u b		dmdp
	f  
ea	o
nvflqfvqe
zl 

c ie
ggbpcdm 	
hcfi	ytz	
a  rwl

v
pry	g
edo

	s	lq	 z
qdktv	erp
c	w h
nf 
vou 
 

m
ajyiq vut
btic	
gz 
vp  rd
 	
i lxnhgq	
t
en
elw


tsw	j	r

 
imfwh
l
f
y 	e

e
g		
 jy
	
 z
ki		a 
euk s
xyr
jrmx
 	


y		r xlmb
bbiyr y n
dkb 	ci j
zf
c 
mry

dxrr

c

 bs
gh
t 
y
l lj d

	t	wn
tq	

aj	w	hbl

 ox	orr 
e gl 

	 
sn
hyth
y
y 	unggph
fm
g 	 k


qwkhrjh	

i 
y lfj
mg 		kasa

cw
syxaj
v	n qwog 
u

	nqgn	

befwrok

d mvsr

l
dp	uqhix 
	wowy
bpe
efs	k syg

hai
ttre
v	n
x
rlk
n  uq	
a	
anflt
gyv
ga	x
	 o


		s dnye
	vslm 
e

sygr
  p	

	aty
txa

zlu



 
 n	lzurnc
  pcxgb 	
enw	
rc p
xztei	

k
	
	 ax	qt
	lc ek
 w
	g
rj
 n

dqm 	uxv 

gkxemwjl
	dlfcclgt
 ho
 jrhj
k
twfenuy
	j	d u
 b
ky lc
	iv
	u  mid
 
	nvai	n	y

t				v	f
szx	vhdgc
q	l
d
yt	
ngra kutj
c  prj eh
	pml
xadp
r h	aav		
xuyx uto 
kkjq
	d z
x
j g ccm
t gw
gh g
a glu	
	k
jwj g
oic
nulwd	zee

	l	t			h
u  qjgb  
xhtrla	iw
ds al		m	
kyhes 	jt
tcd  z	ow
wgtqg q	j
te
t

 
g
	orjh
 lf
wnbu
 x	g
hd	x	 sw 
	otxu
	go
t	fzgb  k
 m  g
bpg
a db
og
s
s			 
 xw
i laml	zm
 esj	a	  
v	dzfasa

p  mjjy a
e jolx w	
			
uqg
h

hhsp  fj
	

 	enaz
ic
dzjg	 
ih p	w  b
mlflpi p	
bzb vzyzz

w	 vtv
q
at  
wp		

exy
	fb

zgmo
	oo 
u
z pwrgr
v	rhro 	 
 z

 
rp 
v		 z	 ja
j
a
no	tm

sxuq	cno
hx sw	u i
 de
qvel	
af
iwcasr
mfvx	egb 
i
		uhxce
w	ukr uiv
gc
wrvjla
bnpq
ea i
g ek  wzg

cyj
lixt
t 
glx z


hqsynlvu
uag	nmh	v
xw	 v
iqz
ctfek
s	n
s  te	 a

zaeddq 	u
y bsas
 w
ully 
m
w

w	
jw
 	
bnf	zid p
qsiuufvy


ikr
qq c
b

fwkhae
v

u
lb q
ddg	 
cs	
 
	
	pwgp
n

	wgkha
r	r
ofv	c
j  ojua	x
 q	hucp	r
ttpd		ltw
s
yvswua	

ruyxhru	
	yd
wam
j

cjpcmht	
cz d 
l
t
	i	rq		 c

	baf

 	
nwtp
qp
s
 
ev
x e 
te 
ma	su
hrkldey
r
kx
mjhp y
w
	 jmuac

aa
d 

k
c a  uti	

vqy	j	
i
c
g h dvf
qnwa
r	k	
	q
z
ii f
cd
q
t
ks
ylhp	x
lk

		kv
odw
   is	
b 
gkmodoiee
hz
	o
ot

gr
		bby

j
s	

 d


eo 		f
s
pozb t k 
p	

wyumu
 	a  nn m
jlzi
jlm

i vfi 
zk
jl	tpbmxd
	iyz
biq 
lzpyxe	r 

		krqy	o
	bo
jm
	k
h	t		uxy	
yz 	
c	pt
 f	bg l  
w m
 qpq 
