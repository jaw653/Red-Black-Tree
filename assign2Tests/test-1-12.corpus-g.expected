Words/Phrases: 2483
Nodes: 1287
Minimum depth: 2
Maximum depth: 23
